storage/innobase/handler/ha_innodb.cc

```diff
@@ -8160,6 +8160,16 @@ ha_innobase::write_row(
 		++trx->will_lock;
 	}
 
+#ifdef WITH_WSREP
+	if (wsrep_thd_get_split(m_user_thd))
+	{
+		/* Note that this transaction is still active. */
+		trx_register_for_2pc(m_prebuilt->trx);
+		/* We will need an IX lock on the destination table. */
+		m_prebuilt->sql_stat_start = TRUE;
+	}
+#endif /* WITH_WSREP */
+
```

Comment:
Can you come up with a more descriptive name?
```c
if (wsrep_is_load_multi_commit(m_user_thd)) {
```
In InnoDB, there is no line break between `)` and `{`.

Reply:
all changes in xtradb are removed from patch